static/app/views/settings/account/accountIdentities.tsx

```diff
@@ -91,4 +192,54 @@ const IdentityPanelItem = styled(PanelItem)`
   justify-content: space-between;
 `;
 
+const InternalContainer = styled('div')`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+`;
+
+const IdentityText = styled('div')<{isSingleLine?: boolean}>`
+  height: 36px;
+  display: flex;
+  flex-direction: column;
+  justify-content: ${p => (p.isSingleLine ? 'center' : 'space-between')};
+  margin-left: ${space(1.5)};
+`;
+const IdentityName = styled('div')`
+  font-weight: bold;
+`;
+const IdentityDateTime = styled(DateTime)`
+  font-size: ${p => p.theme.fontSizeRelativeSmall};
+  color: ${p => p.theme.gray300};
```

Comment:
Avoid direct color usages if you can, otherwise dark mode will not be happy. Do you want `theme.subText`?

Reply:
@leedongwei?

Reply:
```suggestion
  color: ${p => p.theme.subText};
```